locksmith/src/controllers/hookController.ts

```diff
@@ -0,0 +1,42 @@
+// eslint-disable-next-line import/no-unresolved
+import { Response } from 'express-serve-static-core'
+import { networks } from '@unlock-protocol/networks'
+import { SignedRequest } from '../types'
+import { Hook } from '../models'
+
+export function createHookHandler(topic: string) {
+  return async (req: SignedRequest, res: Response): Promise<any> => {
+    const { network, lock } = req.params
+    if (!networks[network]) {
+      return res.status(404).send({
+        error: `Network ${network} not supported!`,
+      })
+    }
+
+    const { callback, lease_seconds, secret, mode } = req.body
```

Comment:
Nit: If I remember the websub spec these should actually be `hub.*`  Let's remain compatible with it please?

Reply:
Okay